### PR TITLE
Add gateway antenna placement

### PR DESCRIPTION
**Pull request by johanstokking**
#### Summary

Add gateway antenna placement.

I will file a follow up issue for Console support.

#### Changes

Add new field to `GatewayAntenna` with the placement. This is reported to Packet Broker Mapper. We had this in V2 and it turned out to be quite useful for site survey purposes.

Also removed some obsolete unused messages from the API to create space for gateways in the device repo.

#### Testing

Local testing

##### Regressions

None expected

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by kschiffer — APPROVED** [review]
Codeowned LGTM

**Review by adriansmares — APPROVED** [review]

File: pkg/identityserver/store/gateway_antenna.go [hunk 1]
```diff
@@ -29,6 +29,8 @@ type GatewayAntenna struct {
 	Gain float32
 
 	Location
+
+	Placement int
```
Comment (htdvisser) [inline pkg/identityserver/store/gateway_antenna.go:33]: We prefer to not do any database migrations in patches, so unless this is critical I would rather see this go to 3.14.

**Review by htdvisser — APPROVED** [review]

File: pkg/identityserver/store/gateway_antenna.go (hunk 1 above)
Comment (johanstokking) [reply pkg/identityserver/store/gateway_antenna.go:33]: Please see target branch

Comment (johanstokking) [reply pkg/identityserver/store/gateway_antenna.go:33]: Milestone was wrong, fixed that

**Pull request merged**